src/curated-corpus/pages/ProspectingPage/ProspectingPage.tsx

```diff
@@ -1,9 +1,11 @@
 import React, { useEffect, useState } from 'react';
 import { DateTime } from 'luxon';
-import { Box, Button, Grid, Hidden } from '@material-ui/core';
-import RefreshIcon from '@material-ui/icons/Refresh';
-import FilterListIcon from '@material-ui/icons/FilterList';
-import AddIcon from '@material-ui/icons/Add';
+import { Box, Button, Grid, Hidden } from '@mui/material';
+import Refresh from '@mui/icons-material/Refresh';
+import FilterList from '@mui/icons-material/FilterList';
+import AddIcon from '@mui/icons-material/Add';
```

Comment:
I see you've gone back to individual imports, but the component names don't have `Icon` in them?

Reply:
Yep, good catch. I've updated the import names!